tests/plugins/identity/user.py

```diff
@@ -0,0 +1,71 @@
+from datetime import datetime
+from typing import Callable, Protocol, TypeAlias, TypedDict, Unpack, final
+
+import pytest
+from mimesis.locales import Locale
+from mimesis.schema import Field, Schema
+
+from server.apps.identity.models import User
+
+
+class UserData(TypedDict, total=False):
+    email: str
+    first_name: str
+    last_name: str
+    date_of_birth: datetime
+    address: str
+    job_title: str
+    phone: str
+    phone_type: int
+
+
+UserAssertion: TypeAlias = Callable[[str, UserData], None]
+
+
+@final
+class RegistrationData(UserData, total=False):
+    password1: str
+    password2: str
+
+
+@final
+class RegistrationDataFactory(Protocol):  # type: ignore[misc]
+    def __call__(self, **fields: Unpack[RegistrationData]) -> RegistrationData:
+        """User registration data factory."""
+
+
+@pytest.fixture(scope="session")
```

Comment:
polish: Please use either single or double quotes consistently. 

Linters typically flag such inconsistencies. I noticed you used single quotes below. Plugins can assist with this, and you can even set up auto-formatting on save to automate this.

There are many minor improvements in the code quality that can be made automatically using linters at no extra cost.

```suggestion
@pytest.fixture(scope='session')
```